src/Bem.js

```diff
@@ -172,11 +172,11 @@ export default function({ preset, naming }) {
         displayName : 'Bem',
 
         childContextTypes : {
-            bemBlock : PropTypes.string
+            bemBlock : () => null
         },
 
         contextTypes : {
-            bemBlock : PropTypes.string
+            bemBlock : () => null
```

Comment:
`const PropTypesStub = () => null;` in the beginning will be more obvious )

Reply:
Maybe...

Reply:
my vote for inline comment in code, without unnecessary constant